me.go

```diff
@@ -0,0 +1,66 @@
+package aiven
+
+import (
+	"context"
+)
+
+type (
+	// UserProfileHandler is the client which interacts with the Current session's user
+	// on Aiven.
+	UserProfileHandler struct {
+		client *Client
+	}
+
+	CurrentUserProfileResponse struct {
+		APIResponse
+		User User `json:"user,omitempty"`
+	}
+
+	User struct {
+		Auth                   []string               `json:"auth,omitempty"`
+		City                   *string                `json:"city,omitempty"`
+		Country                string                 `json:"country,omitempty"`
+		CreateTime             string                 `json:"create_time,omitempty"`
+		Department             *string                `json:"department,omitempty"`
+		Features               Features               `json:"features,omitempty"`
+		Intercom               Intercom               `json:"intercom,omitempty"`
```

Comment:
1. Please use a pointer for `Features`
2. It might have more fields, as it has no strict schema
3. `Intercom` is required field